### PR TITLE
Create message.js file and modify index.html

### PR DESCRIPTION
**Pull request by sebagabs**
Create the `message.js` script file and add it in the `index.html` file. Somehow this branch got a commit of create `RandomStringServlet.java` file, but I had done that on another PR; don't know what happened there. 

These lines are place holders, I plan to modify them. As of right now, I don't understand a lot of what is going on, so I wanna read about a bit more.

### CODE REVIEW

**Comment by SamRod33** [comment]
The only thing I can think of about the accidental commit to this branch with `RandomStringServelt.java` is if you were on this branch and pushed the file. Always do a `git status` before committing helps me know what I am controlling. Hope this helps!

**Comment by GoogleyGabriel** [comment]
> The only thing I can think of about the accidental commit to this branch with `RandomStringServelt.java` is if you were on this branch and pushed the file. Always do a `git status` before committing helps me know what I am controlling. Hope this helps!

+1 to what Sam said, all commits in a branch will show up in a PR. A PR pulls a branch into master, and that includes all commits made in that branch. If you want to avoid this, make a branch off of the branch.

**Review by GoogleyGabriel — APPROVED** [review]
Looking great! This is exactly how PRs should be -- keep up the hard work :)

File: portfolio/src/main/java/com/google/sps/servlets/RandomStringServlet.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/tester")
```
Comment (GoogleyGabriel) [inline portfolio/src/main/java/com/google/sps/servlets/RandomStringServlet.java]: Change this to a more descriptive name, such as "fetch-hello-world-string".

Comment (sebagabs) [reply portfolio/src/main/java/com/google/sps/servlets/RandomStringServlet.java]: Just did so in a commit.

**Comment by sebagabs** [comment]
I made some final commits for this pull request. Just changed some details @GoogleyGabriel pointed out, and modified the `script.js` file to include the function from `message.js`; got this I idea from Claudia's code. Also, I changed the `index.html` file so that it would not have a reference to `message.js` since now it does not exist.

**Review by GoogleyGabriel — APPROVED** [review]

**Pull request merged**